examples/isometric/main.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build example
+// +build example
+
+package main
+
+import (
+	"log"
+
+	"github.com/hajimehoshi/ebiten/v2"
+)
+
+func main() {
+	ebiten.SetWindowTitle("Isometric (Ebiten Demo)")
+	ebiten.SetWindowSize(640, 480)
+	ebiten.SetWindowResizable(true)
+	ebiten.SetMaxTPS(144)
```

Comment:
Is 144 intended? If so, why?

Reply:
This ensures 144Hz displays are satisfied, as lower TPS can cause duplicated frames and higher TPS would not result in any perceptual difference on higher Hz displays.

Reply:
I don't think this example is so FPS sensitive that this has to have 144 Hz. For consistency with other examples, would it be possible to remove this? Thanks,